incubator/hydra/templates/hydra_deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
@20zinnm yeah I agree with that, but at the same time most of the k8s deployments were seeing right now are utilizing RDS or some hosted datastore, so I would hate to marry it either. Kind of a rock and a hard place